src/BenchmarksApps/BasicMinimalApi/Program.cs

```diff
@@ -2,7 +2,7 @@
 using BasicMinimalApi;
 
 var builder = WebApplication.CreateSlimBuilder(args);
-builder.Logging.ClearProviders(); // Clearing for benchmark scenario, template has AddConsole();
+// No logging for benchmark scenario, template has AddConsole();
```

Comment:
So remove the explicit "Application started" output at the end of the app?

However ... not logging and not having logging provider is not the same in terms of perfs. Having logging provider registered has some perf impact because there will be custom logging scopes created still and some using of async locals. You should measure the differences for knowledge.

Reply:
`ClearProviders()` calls `services.RemoveAll<ILoggerProvider>()`.

https://github.com/dotnet/runtime/blob/75eac9643f3d8952df3cffd928e7987887bb7dab/src/libraries/Microsoft.Extensions.Logging/src/LoggingBuilderExtensions.cs#L46-L50

When you call `CreateSlimBuilder`, there are no `ILoggerProvider` services registered. So there are no logging providers to remove:

![image](https://user-images.githubusercontent.com/8291187/220515338-0d0e0427-40fc-4e71-8e0f-3855778c139b.png)

So calling `ClearProviders()` doesn't do anything here.

Here's another way to prove it:

![image](https://user-images.githubusercontent.com/8291187/220515609-cac5a6e3-b0a9-45d0-81f7-9a0710b2177a.png)

The number of services before and after calling `builder.Logging.ClearProviders()` is the same.